src/Sulu/Bundle/ContactBundle/Resources/config/lists/contacts.xml

```diff
@@ -296,12 +296,13 @@
         </property>
 
         <group-concat-property
-            name="tagIds"
-            visibility="never"
+            name="tagNames"
+            visibility="yes"
```

Comment:
should be not visible by default should be set to `no` and the user can then enable it when they use it.